frontend/webapp/core/modules.py

```diff
@@ -113,3 +142,25 @@ def get_response_all(self):
 
     def get_response_formatted_all(self):
         return self.get_response_all()
+
+    def update_intended_process_states(self, name, running=False):
+        try:
+            access_token = create_access_token(identity=current_user)
+            graqphql_request_headers = {
+                "Content-Type": "application/json; charset=utf-8",
+                "Authorization": "Bearer {}".format(access_token),
+            }
+            graqphql_request_payload = json.dumps(
```

Comment:
there is a jsonify function already on flask that can be used for requests.

Reply:
it works this was, with jsonify it throws errors. Since we need to start closing this PR, I suggest if you want to make an issue like "jsonify all Flask requests".